modules/ROOT/pages/provisioning-hyperv.adoc

```diff
@@ -0,0 +1,104 @@
+= Provisioning Fedora CoreOS on Microsoft HyperV
+
+This guide shows how to provision new Fedora CoreOS (FCOS) nodes on Microsoft's HyperV.
+
+== Prerequisites
+
+You must have an Ignition configuration file containing your customizations. If you do not have one, see xref:producing-ign.adoc[Producing an Ignition File]
+You will also need a small utility from the libhvee https://github.com/containers/libhvee called `kvpctl`. It "injects" your Ignition file into your virtual machine. Precompiled binaries can be found on the project's https://github.com/containers/libhvee/releases[release page].
+
+=== Downloading the HDX image
+
+HDX images can be manually downloaded from the download page.  Be sure to decompress the image after downloading.
+
+== Booting a new VM on Microsoft HyperV
+
+=== Creating a virtual machine
+
+There are numerous ways to create a virtual machine in HyperV.  This approach uses the HyperV Manager and its new virtual machine wizard. Once you have the HyperV Manager graphic interface running, right-click on _New_ as seen in the following image.
+
+image::new_hyperv.png[HyperV New Virtual Machine Wizard]
```

Comment:
```suggestion
image::new_hyperv.png[Hyper-V Manager]
```

As bgilbert pointed out the official spelling is Hyper-V; I think we should be consistent. Also I made a mistake in my earlier suggestion: the image is actually of the manager UI and not the wizard interface.